src/index.css

```
@@ -99,6 +99,34 @@ body {
+  arrive. Tint the cover box so the grid reads as a shelf of books immediately, and fade each
+  cover in over it once decoded rather than snapping.
+*/
+.book-cover-frame {
+  background-color: var(--muted);
+}
+
+.book-cover {
+  animation: bookCoverIn 200ms ease-out;
+}
+
+@keyframes bookCoverIn {
```

Stylelint (17.14.1)

[error] Expected keyframe name "bookCoverIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)